### ISSUE TITLE
运行demo显示推理不正常

### ISSUE THREAD

**Issue by Elton-Yang** [infinigence/Infini-Megrez-Omni#15]
按照说明步骤启动demo，显示了一个警告，结果也不正常：
$ python example_chat_hf.py
Loading checkpoint shards: 100%|█████████████████████████████████████| 2/2 [00:00<00:00,  5.08it/s]
/pub/venv_1/lib/python3.12/site-packages/transformers/models/auto/image_processing_auto.py:524: FutureWarning: The image_processor_class argument is deprecated and will be removed in v4.42. Please use `slow_image_processor_class`, or `fast_image_processor_class` instead
  warnings.warn(
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Hello! It looks like you might have accidentally sent a series of exclamation marks. How can I assist you today?<|turn_end|>

**Comment by lizhiyuanUSTC** [comment]
可以在[这儿](https://github.com/infinigence/Infini-Megrez-Omni/blob/main/example_chat_hf.py#L26)修改您想针对图片做的提问
